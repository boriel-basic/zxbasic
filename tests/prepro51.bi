#define X A(1)
#define A(x) x

X



