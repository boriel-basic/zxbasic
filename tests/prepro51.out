#line 1 "prepro51.bi"


A(1)



